Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM yastdevel/ruby:sle12-sp3
+RUN zypper --non-interactive update yast2
```

Comment:
I introduced this because it would not install the latest version from there. I couldn't understand why, but Travis would only become green with this. I'll try again, though.

Reply:
No, it does not work without that: It will only get yast2-3.2.40, not -3.2.41 which contains the required changes, so the testsuite (`make check`) fails in Travis with a return code 16 for
- autoyast
- readwrite
- readwrite2

That was actually one thing that took me so long to track down. I really don't know why it would pick up yast2-3.2.40 (which is also not in Leap AFAIK), but not -3.2.41, but that's the way it behaves.

I know it's not pretty to have one single package update in the Dockerfile, but OTOH it should not hurt either.

Reply:
Leap 42.3 has yast2-3.2.36 AFAICS:
https://build.opensuse.org/package/show/openSUSE:Leap:42.3/yast2

Reply:
(Leap 42.3 is the base Docker image we use for SLE-12-SP3 in Travis)